typegraph/deno/src/effects.ts

```diff
@@ -23,3 +23,9 @@ export function delete_(idempotent = true): EffectDelete {
 export function update(idempotent = true): EffectUpdate {
   return { tag: "update", val: idempotent };
 }
+
+export const effectPrefix = "__effect_";
```

Comment:
I believe we shall support simple string as well so I would avoid any prefix

Reply:
If we allow simple string then we have to reserve `update`, `delete`, `create`, and `none` for the typescript frontend. 
We can however have a workaround by making `PerEffect` explicit
```typescript
struct({ a: t.string() })
.set({ [NONE]: {a: 1}, [CREATE]: {a: 2} })  // prefix is used to distinguish between an object and a per effect object
```
becomes
```typescript
struct({ a: t.string() })
.set({ none: {a: 1}, create: {a: 2} }, "PerEffect")
```

Reply:
We can use [symbols](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol).